thing-management.md

```diff
@@ -49,47 +49,33 @@ Object containing nodeId and status.
 
 > `status` can be:
 - `not_in_network`: A device is not currently included in the network.
-- `included`: A device is included in the network and is functional.
-- `failing`: A device is included in the network but is now marked as failing (e.g. communication fails or it has not woken up for longer than expected)
+- `added`: A device is included in the network and is functional.
+- `failed`: A device is included in the network but is now marked as failing (e.g. communication fails or it has not woken up for longer than expected)
 > `nodeId`: Unique identifier assigned to a device by a Smarthub.
 
 #### `generic_device_class`
 
-Z-Wave generic device class.
-
-Examples:
-
-GENERIC_TYPE_ENTRY_CONTROL, GENERIC_TYPE_SWITCH_BINARY, GENERIC_TYPE_SWITCH_MULTILEVEL.
+Z-Wave generic device class (represented by a number).
 
 #### `specific_device_class`
 
-Z-Wave specific device class. Along with generic device type creates a specific information about device type.
-
-Examples:
-
-SPECIFIC_TYPE_COLOR_TUNABLE_BINARY, SPECIFIC_TYPE_DOOR_LOCK, SPECIFIC_TYPE_CLASS_A_MOTOR_CONTROL
-
-Example of generic and specific device class pair and a matching meaning:
-
-
-GENERIC_TYPE_SWITCH_MULTILEVEL and SPECIFIC_TYPE_CLASS_A_MOTOR_CONTROL means Z-Wave Window Covering device type.
-GENERIC_TYPE_SWITCH_MULTILEVEL and SPECIFIC_TYPE_COLOR_TUNABLE_MULTILEVEL means Z-Wave Color Switch device type.
+Z-Wave specific device class. Along with generic device type creates a specific information about device type (represented by a number).
 
 #### `installer_icon_type`
 
-Z-Wave specification has a list of icons with hexadecimal values corresponded to every one of them.
+Z-Wave specification has a list of icons with decimal values corresponded to every one of them.
 
 #### `manufacturer_id`
 
-Unique value given for every manufacturer who sells Z-Wave devices. 
+Unique decimal value given for every manufacturer who sells Z-Wave devices. 
```

Comment:
what is decimal value? integer value maybe ?

Reply:
yes

Reply:
fixed